query-connector/src/app/queryBuilding/buildFromTemplates/page.tsx

```diff
@@ -95,100 +201,111 @@ export default function QueryTemplateSelection() {
     });
   };
 
-  const submitForm = () => {
-    // TODO: do something with selectedConditions on next step/page
-    // will be addressed in https://linear.app/skylight-cdc/issue/QUE-65/create-the-valueset-selection-page
-    console.log(selectedConditions);
-  };
-
   const atLeastOneItemSelected =
     selectedConditions && Object.values(selectedConditions).length > 0;
 
   return (
     <>
       <SiteAlert />
-      <div className="main-container__wide">
+      <div className={classNames("main-container__wide", styles.mainContainer)}>
         <Backlink
           onClick={() => {
-            router.push("/queryBuilding");
+            // TODO: this can be tidied up...
```

Comment:
Do we need to action/make tickets for these TODO's at all? If not, can we delete these?

Reply:
I'll make a follow up ticket (also for @robertandremitchell's suggestion above so we can merge this and unblock the drawer ticket)